packages/report-server/src/routes/FetchTransformSchemaRoute.ts

```diff
@@ -20,15 +20,32 @@ export type FetchTransformSchemaRequest = Request<
 
 export class FetchTransformSchemaRoute extends Route<FetchTransformSchemaRequest> {
   async buildResponse() {
+    const removeRedundantConfigs = (fields: Record<string, any | any[]> | any[]) => {
+      if (typeof fields !== 'object' || Array.isArray(fields)) {
+        return fields;
+      }
+      const updatedFields = { ...fields };
+      Object.entries(fields).forEach(([key, value]) => {
+        if (typeof value === 'object') {
+          updatedFields[key] = removeRedundantConfigs(value);
+        }
+        const isEmptyArray = Array.isArray(value) && value.length === 0;
+        if (isEmptyArray) {
+          delete updatedFields[key];
+        }
+      });
+      return updatedFields;
+    };
```

Comment:
Remove schema with empty oneOf array. It will cause error in `json-editor` validation.